contrib/debian/control

```diff
@@ -48,11 +48,9 @@ Pre-Depends: adduser,
              dpkg (>= 1.17.14),
              libcap2-bin (>=1:2.0),
              lsb-base (>= 3.1-23.2)
-Conflicts: netdata-core,
-           netdata-plugins-bash,
+Conflicts: netdata-plugins-bash,
            netdata-plugins-python,
            netdata-web,
```

Comment:
Should we remove a comma here because `netdata-web` is the last item after this change?